assignments/arrays.js

```diff
@@ -66,31 +66,51 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
 
 
+console.log('Car 33 is a ' + inventory[32].car_year + ' ' + inventory[32].car_make + ' ' + inventory[32].car_model);
+
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
 let lastCar = 0;
-console.log();
-
+console.log('Make: ' + inventory[49]['car_make'] + ' Model: ' + inventory[49]['car_model']);
```

Comment:
A more dynamic approach would be to access the last car using `lastCar = inventory[inventory.length - 1]` in case the dealer adds more cars to their inventory